tests/test_notebooks.py

```diff
@@ -43,6 +43,35 @@ def test_eln_import(selenium_driver):
     driver.find_element(By.ID, "tooltip")
 
 
-def test_computational_resources(selenium_driver):
+def test_computational_resources(selenium_driver, aiidalab_exec):
+    """Test the quicksetup of the code"""
+    # check the code pw-7.0 is not in code list
+    output = aiidalab_exec("verdi code list").decode().strip()
+    assert "pw-7.0" not in output
+
     driver = selenium_driver("notebooks/computational_resources.ipynb")
-    driver.find_element(By.XPATH, '//button[text()="Setup new code"]')
+
+    # click the "Setup new code" button
+    driver.find_element(By.XPATH, '//button[text()="Setup new code"]').click()
+
+    # Select daint.cscs.ch domain
+    driver.find_element(By.XPATH, '//option[text()="daint.cscs.ch"]').click()
+
+    # Select computer multicore
+    driver.find_element(By.XPATH, '//option[text()="multicore"]').click()
+
+    # select code pw-7.0-multicore
+    driver.find_element(By.XPATH, '//option[text()="pw-7.0-multicore"]').click()
+
+    # fill the SSH username
+    driver.find_element(
+        By.XPATH, "//label[text()='SSH username:']/following-sibling::input"
+    ).send_keys("dummyuser")
+
+    # click the quick setup
+    driver.find_element(By.XPATH, '//button[text()="Quick Setup"]').click()
+    driver.implicitly_wait(1.0)
```

Comment:
I don't think `implicitly_wait` does what you need here.
https://selenium-python.readthedocs.io/waits.html#implicit-waits

```suggestion
    time.sleep(1.0)
```

Reply:
However, I still do not quite understand the difference between their documentation. But I think you are right. 
We need anyway move to a more efficient "expected condition" wait.